external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -182,10 +182,16 @@
           SkipUnchangedFiles="true"/>
   </Target>
 
+<!--
+  
+  Disable this check until we get the CoreCLR and TFS versions of System.Private.CoreLib back in sync after recent breaking changes.
```

Comment:
I'm not sure I understand why do we want to comment this out. We should instead just update both of the PN packages to the same version. Is there not a new TestILC from the same build available?

Reply:
We turned the targeting pack off a few days ago but not TestIlc - not realizing this check exists. So they won't be back in sync until we get a UAP build out of CoreFX.